Client/Ecosia/UI/EmptyBookmarksView.swift

```diff
@@ -0,0 +1,179 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+
+final class EmptyBookmarksView: UIView, NotificationThemeable {
+
+    required init?(coder: NSCoder) {
+        assertionFailure("This view is only supposed to be instantiated programmatically")
+        return nil
+    }
+    
+    private let titleLabel: UILabel = {
+        let label = UILabel()
+        label.text = .localized(.noBookmarksYet)
+        label.textAlignment = .center
+        label.font = UIFontMetrics(forTextStyle: .body).scaledFont(for: .systemFont(ofSize: 17, weight: .semibold))
```

Comment:
I've noticed that the codebase utilizes some `UX` wrapper at the top of the classes to put every hardocded dimension / UI-related stuff in one place to facilitate updates if needed.
There may be other places where the same suggestion may come up so I won't comment there too 👍 .

Reply:
Good catch, thx. Done.